keps/sig-apps/20190318-PodDisruptionBudget-graduation-to-stable.md

```diff
@@ -80,6 +80,60 @@ modifications and use the updated values when making decisions.
 
 This feature is implemented by [this PR](https://github.com/kubernetes/kubernetes/pull/69867).
 
```

Comment:
these two sections are good to include, but the current text makes it unclear if the proposal is accepted. I would suggest:
1. merge a quick update that just adds the two sections describing the problems, with placeholders for the solutions
2. open an update to each section that describes the proposed solution with language like "To fix this, we will..."

That will let us discuss each proposed solution separately, and once merged, will make it clear to the reader what is agreed on.

Reply:
Sounds good. I have updated this PR to only include the sections describing the problems.